plugins/portcheck/plugin.go

```diff
@@ -59,7 +60,7 @@ func checkAutopeeringConnection() {
 	defer conn.Close()
 
 	// create a new discovery server for the port check
-	disc := discover.New(local.GetInstance(), autopeering.ProtocolVersion, autopeering.NetworkVersion, discover.Logger(log))
+	disc := discover.New(local.GetInstance(), autopeering.ProtocolVersion, config.Node().GetUint32(autopeering.CfgNetworkVersion), discover.Logger(log))
```

Comment:
I think it would be cleaner to define a variable `autopeering.NetworkVersion` (or even function) then you don't need to read from the config here.